_includes/manuals/2.2/3.2.2_installer_options.md

```diff
@@ -177,6 +181,10 @@ More information about compute resources can be found in the [Compute Resources
         <td style='white-space:nowrap'>--[no-]enable-foreman-plugin-snapshot-management</td>
         <td>Enable 'foreman_plugin_snapshot_management' puppet module (<a href="https://github.com/ATIX-AG/foreman_snapshot_management">foreman_snapshot_management</a>)</td>
       </tr>
+      <tr>
+        <td style='white-space:nowrap'>--[no-]enable-foreman-plugin-statistics</td>
+        <td>Enable 'foreman_plugin_statistics' puppet module (<a href="/plugins/foreman_statistics">foreman_statistics</a>)</td>
```

Comment:
I'm now remembering I wrote the code for this and happy that it still works. Note how it's the Foreman manual vs the GH project :)